python/api/src/araali/__main__.py

```diff
@@ -510,6 +511,52 @@ def fw_config(args):
     knobs = api.API().get_fw_config(zone=args.zone, tenant=args.tenant)
     print(yaml.dump(knobs))
 
+def quickstart(args):
+    if args.quickstart_subparser_name == "vm":
+        success, result = _aws.cf_launch_fortified_vm(stack_name="araali-quickstart-vm-stack", email=args.email, key_pair=args.key)
+        if not success:
+            print("Quickstart setup failed")
+            return
+        return result
+    elif args.quickstart_subparser_name == "eks":
+        EKS_STACK_NAME = "araali-quickstart-stack"
+        EKS_CLUSTER_NAME = "araali-quickstart-eks"
+        cluster_name = args.name
+        launch_cluster = args.launchcluster
+        if cluster_name == "":
```

Comment:
you can query aws to find if the cluster is existing

Reply:
Doesn't look like you have addressed this. You need to do one of two things:

- make the name unique by adding a timestamp to the name. OR
- fail the command if the name already exists and ask user to choose a different name

Reply:
If the name already exists, I am just doing a helm install on the provided cluster without launching one. If it doesn't, then I am creating a new one. If the user hasn't specified a name, then I use the default name with timestamp.